resources/operator.yaml

```diff
@@ -17,7 +17,7 @@ spec:
       containers:
         - name: benchmark-operator
           # Replace this with the built image name
-          image: quay.io/benchmark-operator/benchmark-operator:latest
+          image: "quay.io/nichawla/benchmark_op_nikhil:testing"
```

Comment:
Let's keep this using the main image.

Reply:
yep, when the PR is ready update it to master as it should be after the rebase.